scheduler/src/cook/mesos/task.clj

```diff
@@ -297,6 +309,9 @@
                                   (if (:network docker)
                                     (update docker :network cook-network->mesomatic-network)
                                     docker)))
+                        (update :docker
+                                #(clojure.set/rename-keys % {:port-mapping :port-mappings}))
```

Comment:
Can we request `clojure.set :as set` and use `set/rename-keys` here.

Random rant: why is this method in `clojure.set`? https://dev.clojure.org/jira/browse/CLJ-1820

Reply:
Done